airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/common.py

```diff
@@ -330,23 +330,9 @@ def default_params(self) -> STEP_PARAMS:
             "--durations": ["3"],  # Show the 3 slowest tests in the report
         }
 
-    def regression_tests_command(self) -> List[str]:
-        """
-        This command:
-
-        1. Starts a Google Cloud SQL proxy running on localhost, which is used by the connection-retriever to connect to postgres.
-        2. Gets the PID of the proxy so it can be killed once done.
-        3. Runs the regression tests.
-        4. Kills the proxy, and waits for it to exit.
-        5. Exits with the regression tests' exit code.
-        We need to explicitly kill the proxy in order to allow the GitHub Action to exit.
-        An alternative that we can consider is to run the proxy as a separate service.
-
-        (See https://docs.dagger.io/manuals/developer/python/328492/services/ and https://cloud.google.com/sql/docs/postgres/sql-proxy#cloud-sql-auth-proxy-docker-image)
-        """
-        run_proxy = "./cloud-sql-proxy prod-ab-cloud-proj:us-west3:prod-pgsql-replica --credentials-file /tmp/credentials.json"
+    def _test_command(self) -> str:
```

Comment:
as `should_read_with_state`  or `test_evaluation_mode` attributes are not set in `__init__` I'd suggest making this function static and make the caller pass these parameters.
We have no guarantee that ` self.test_evaluation_mode` is set...